src/Microsoft.IdentityModel.Clients.ActiveDirectory/UserIdentifier.cs

```diff
@@ -61,7 +61,7 @@ public sealed class UserIdentifier
         private const string AnyUserId = "AnyUser";
 
         private static readonly UserIdentifier AnyUserSingleton =
-            new UserIdentifier(AnyUserId, UserIdentifierType.UniqueId);
+            new UserIdentifier(AnyUserId, UserIdentifierType.OptionalDisplayableId);
```

Comment:
it's public, thus it could be used by external parties

---
In reply to: [156450874](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/911#discussion_r156450874) [](ancestors = 156450874)